oci.go

```diff
@@ -245,8 +260,15 @@ func processCgroupsPathForResource(ociSpec oci.CompatOCISpec, resource string) (
 		}
 	}
 
-	if cgroupMountFound == false {
-		return "", fmt.Errorf("cgroupsPath is absolute, cgroup mount MUST exist")
+	if !cgroupMountFound {
+		if isPod {
+			return "", fmt.Errorf("cgroupsPath %q is absolute, cgroup mount MUST exist",
```

Comment:
Sure, as I said, it isn't really part of your PR, but the current wording just doesn't make sense to me at the moment.